wgsl/index.bs

```diff
@@ -115,6 +115,24 @@ div.syntax > p > code {
     "title":"The Unicode Standard, Version 14.0.0",
     "isbn":"978-1-936213-29-0",
     "id":"UnicodeVersion14"
+  },
+  "DeRemer1969": {
+    "href":"http://publications.csail.mit.edu/lcs/pubs/pdf/MIT-LCS-TR-065.pdf",
+    "author":"F. L. DeRemer",
+    "publisher":"Massachusetts Institute of Technology",
+    "date":"24 October 1969",
+    "title":"Practical Translators for LR(k) Languages"
+  },
+  "VanWyk2007": {
+    "href":"https://dl.acm.org/doi/10.1145/1289971.1289983",
+    "title":"Context-Aware Scanning for Parsing Extensible Languages",
+    "authors": [
+      "Eric R. Van Wyk",
+      "August C. and Schwerdfeger"
```

Comment:
I think the "and" is a typo? When I check the link the author info doesn't include it.
```suggestion
      "August C. Schwerdfeger"
```

Reply:
I badly translated the Bibtex info from the ACM site. Thanks!